### DISCUSSION TITLE
diverse Fahrzeuge fahren mit CP nur noch 8 km/h

### DISCUSSION THREAD

**Discussion by Bud3ncer** [Courseplay/Courseplay_FS22#2217] [Q&A]
Hallo zusammen

Meine Traktoren fahren teils nach dem Wenden nur noch mit 8 km/h übers Feld obwohl das Werkzeug 15 km/h schaffen würde. Die PS-Zahl passt respektive ist höher als das Werkzeug fordert. Wo mache ich den fehler?

**Comment by Tensuko** [comment]
Es fehlen so viele Infos....
CP Version.
Welches Fahrzeug.
Geht es da berg auf?
Usw usw.

**Reply by Bud3ncer** [reply]
aktuellste CP Version 7.1.2.5, ist aber auch schon bei 7.1.2.4 gewesen
Multimap Feld 23, eben
Es spielt keine Rolle welches Fahrzeug. Ich habe den Verdacht, wenn es kein stufenloses Getriebe ist, dann hat er bei mehren aktiven Fahrzeugen Probleme dies zu handhaben/berechnen.

**Reply by Tensuko** [reply]
Aha, das ist aber ein normales LS Problem und findet nicht nur mit CP statt sondern auch durch spieler bewegte Fahrzeuge statt.

**Reply by Bud3ncer** [reply]
ah ok, danke... wusste ich nicht

**Reply by Tensuko** [reply]
Man kann ja auch nicht alles wissen :)
Ich weis nicht ob Giants da mal etwas nachbessert, wenigstens für KI betriebene Fahrzeuge.
